Prevent reseting DLM Dashboard integration checkbox

jalbador opened the pull request:
Every time you configure/edit a job in Jenkins, it resets the "DLM Dashboard integration" checkbox to enabled. It should be set and remember the option configured,.

![image](https://cloud.githubusercontent.com/assets/5928311/14670376/d2969182-06e4-11e6-9009-22a3d967c690.png)

JasonCrease committed:
A few typos!

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0

JasonCrease committed:
back to snapshot

JasonCrease committed:
release properties

JasonCrease committed:
removed tag

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0

JasonCrease committed:
Revert "[maven-release-plugin] prepare release redgate-sql-ci-1.0"

This reverts commit 9cb269a61d203d3853515bc60cf24ce236f4ccd2.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0

JasonCrease committed:
Revert "[maven-release-plugin] prepare release redgate-sql-ci-1.0"

This reverts commit 7e2a11c8a100e8aa0a326baeeb3428874fb326c9.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0

JasonCrease committed:
Revert "[maven-release-plugin] prepare release redgate-sql-ci-1.0"

This reverts commit fb5cd5b481362eb0774cb7bb264d6acb78b63ab2.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0

JasonCrease committed:
removed tag from pom

JasonCrease committed:
no release properties

JasonCrease committed:
added snapshot back to pom.xml

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0

JasonCrease committed:
Revert "[maven-release-plugin] prepare release redgate-sql-ci-1.0"

This reverts commit 6e35dd53d866b2b60d92ba3fa28e4826ebdfa3cb.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.1

JasonCrease committed:
[maven-release-plugin] prepare for next development iteration

JasonCrease committed:
Updated install instructions.

JasonCrease committed:
ignore the iml file

JasonCrease committed:
Revert "ignore the iml file"

This reverts commit 928cecccb86099f00b08aee619486c48f08d273c.

JasonCrease committed:
ignore the iml

JasonCrease committed:
Better recent locations for SQL CI.

And break once it's been found.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.2

JasonCrease committed:
try 1.0.2 again

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.2

JasonCrease committed:
retry 1.0.2

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.2

JasonCrease committed:
Rolling forward to 1.0.3, 1.0.2 tag is used

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.3

JasonCrease committed:
[maven-release-plugin] prepare for next development iteration

JasonCrease committed:
using Jenkins 1.634

JasonCrease committed:
Better parameter passing

Quote stuff as necessary.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.4

JasonCrease committed:
Try again on 1.0.5

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.5

JasonCrease committed:
Revert "[maven-release-plugin] prepare release redgate-sql-ci-1.0.5"

This reverts commit 360d01bacd288d5573a9c04eadf4581c28350992.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.5

JasonCrease committed:
Try 1.0.4 again.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.4

JasonCrease committed:
Revert "[maven-release-plugin] prepare release redgate-sql-ci-1.0.4"

This reverts commit 8a35c6355b45595527af972e19bcf0e603c416bb.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.4

JasonCrease committed:
Revert "[maven-release-plugin] prepare release redgate-sql-ci-1.0.4"

This reverts commit f19918513ed1489f8029699d0308e07e91f54af7.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.4

JasonCrease committed:
Revert "[maven-release-plugin] prepare release redgate-sql-ci-1.0.4"

This reverts commit 6fd7d08f8eaf903d102cdec3115e4002748fb602.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.4

JasonCrease committed:
Revert "using Jenkins 1.634"

This reverts commit 4edaebaa8e410bbf937eb3f268c408134209d08c.

JasonCrease committed:
Revert "[maven-release-plugin] prepare release redgate-sql-ci-1.0.4"

This reverts commit 66da6f7c6b5489b7f55c9e843233bc0db8a731c9.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.4

JasonCrease committed:
Revert "[maven-release-plugin] prepare release redgate-sql-ci-1.0.4"

This reverts commit be28422db168b20b8aabb14abad8e0d472fa4480.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.4

JasonCrease committed:
Roll forward to 1.0.5

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.5

JasonCrease committed:
Revert "[maven-release-plugin] prepare release redgate-sql-ci-1.0.5"

This reverts commit dc043c6847a330f055e760182a310fdbf646fec7.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.5

JasonCrease committed:
[maven-release-plugin] prepare for next development iteration

JasonCrease committed:
Added environment variable support

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.6

JasonCrease committed:
[maven-release-plugin] prepare for next development iteration

JasonCrease committed:
Null check to preserve backwards compatibility.

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.7

JasonCrease committed:
[maven-release-plugin] prepare for next development iteration

JasonCrease committed:
Added DLM Dashboard support

JasonCrease committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.8

JasonCrease committed:
[maven-release-plugin] prepare for next development iteration

cjheppell committed:
check dlm dashboard box only if it was set previously

PeterGerrard committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.10

PeterGerrard committed:
[maven-release-plugin] prepare for next development iteration

PeterGerrard committed:
Merge pull request #2 from cjheppell/dlm-dashboard-checkbox

Check dlm dashboard box only if it was set previously

PeterGerrard committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.11

PeterGerrard committed:
[maven-release-plugin] prepare for next development iteration

cjheppell commented:
This has been fixed as of version 1.0.11. See the [Jenkins repository for the SQL CI plugin](https://github.com/jenkinsci/redgate-sql-ci-plugin) for where this issue was fixed.

PeterLGerrard-Redgate committed:
Get the environment variable and check file on the agent.

PeterLGerrard-Redgate committed:
Code review changes, more java-like.

PeterGerrard committed:
Merge pull request #3 from PeterGerrard/DistributedAgents

Handle running on distributed agents

PeterLGerrard-Redgate committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.12

PeterLGerrard-Redgate committed:
[maven-release-plugin] prepare for next development iteration

PeterLGerrard-Redgate committed:
Rename to DLM Automation

PeterLGerrard-Redgate committed:
Set the environment variable for FUR.

PeterLGerrard-Redgate committed:
Revert the folder move

PeterLGerrard-Redgate committed:
Use the renamed page

PeterLGerrard-Redgate committed:
Change the developer in the POM file.

JasonCrease committed:
Merge pull request #6 from PeterGerrard/UpdateDeveloper

Change the developer in the POM file.

cjheppell committed:
Merge pull request #5 from PeterGerrard/Rebranding

Rebrand as DLM Automation

cjheppell committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.13

cjheppell committed:
[maven-release-plugin] prepare for next development iteration

PeterLGerrard-Redgate committed:
Escape the ci exe path.

PeterGerrard committed:
Merge pull request #8 from PeterGerrard/EscapeExePath

Escape the Exe path.

PeterLGerrard-Redgate committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.14

PeterLGerrard-Redgate committed:
[maven-release-plugin] prepare for next development iteration

MatthewFlatt committed:
DLM v2

MatthewFlatt committed:
typo

MatthewFlatt committed:
install directory names

MichelleTaylorRG committed:
Merge pull request #9 from red-gate/DLMv2

DLM v2

MichelleTaylorRG committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.15

MichelleTaylorRG committed:
[maven-release-plugin] prepare for next development iteration

MichelleTaylorRG committed:
Merge remote-tracking branch 'refs/remotes/jenkinsci/master'

MichelleTaylorRG committed:
Update POM

MichelleTaylorRG committed:
Move comment

MichelleTaylorRG committed:
and add an end tag like a non-idiot

MichelleTaylorRG committed:
Null check for checkOutPath

Returns the same as if SQLCI had failed

MichelleTaylorRG committed:
Remove dead store to envVars

MichelleTaylorRG committed:
Use isempty rather than equals emptystring

MichelleTaylorRG committed:
Use string buffer for looping string append

MichelleTaylorRG committed:
Merge pull request #10 from red-gate/pom-INFRA-588-update

INFRA-588: POM update

PeterLGerrard-Redgate committed:
[maven-release-plugin] prepare release redgate-sql-ci-1.0.16

PeterLGerrard-Redgate committed:
[maven-release-plugin] prepare for next development iteration

IvoMiller committed:
Add PowerShell files

IvoMiller committed:
Remove unused code

IvoMiller committed:
Standardise formatting

IvoMiller committed:
Fix IntelliJ warnings

IvoMiller committed:
Move method to different class via automated refactor

IvoMiller committed:
Rename class and make abstract

IvoMiller committed:
Fix more IntelliJ warnings

IvoMiller committed:
Add rudimentary test for abstract class

IvoMiller committed:
Extract method

IvoMiller committed:
Extract method

IvoMiller committed:
Extract method

IvoMiller committed:
Extract method

IvoMiller committed:
Rename method

IvoMiller committed:
Make error logging consistent

IvoMiller committed:
Use Java 8

IvoMiller committed:
Execute new PowerShell

IvoMiller committed:
Always get latest version

To be replaced with UI option in subsequent commit

IvoMiller committed:
Add product version option to UI for all tasks

PeterGerrard committed:
Merge pull request #13 from red-gate/refactor-of-cmdlet-execution-code

Refactor of cmdlet execution code

IvoMiller committed:
Fix population of fields on reconfiguring tasks

PeterGerrard committed:
Merge pull request #14 from red-gate/sql-change-automation-compatibility

Sql change automation compatibility

IvoMiller committed:
Update publish task

IvoMiller committed:
Update sync task

IvoMiller committed:
Use enum for project options in build task

IvoMiller committed:
Extract method and use switch statement

IvoMiller committed:
Add SQL Change Automation project option to build task

IvoMiller committed:
Finish updating build task

IvoMiller committed:
Updating test task

IvoMiller committed:
Update README

IvoMiller committed:
Update pom

IvoMiller committed:
Update task display names

IvoMiller committed:
Update UI descriptions

PeterGerrard committed:
Merge pull request #15 from red-gate/sql-change-automation-project-support

Sql change automation project support

PeterGerrard committed:
Merge pull request #16 from red-gate/rebrand-to-sql-change-automation

Rebrand to sql change automation

IvoMiller committed:
Copy PowerShell files to remote agent workspace before execution

PeterGerrard committed:
Merge pull request #17 from red-gate/fix-powershell-execution-on-remote-agents

Copy PowerShell files to remote agent workspace before execution

IvoMiller committed:
Fix potential null pointer for new test source parameter

PeterGerrard committed:
Merge pull request #18 from red-gate/test-source-upgrade-bugfix

Fix potential null pointer for new test source parameter

Greg-Smulko committed:
Add missing colon

Greg-Smulko committed:
Add missing field title

Greg-Smulko committed:
Add "optional" notion to SQL Data Generator path

PeterGerrard committed:
Merge pull request #19 from red-gate/minor-ui-improvements

Minor UI improvements

PeterLGerrard-Redgate committed:
Fix findbugs issues

PeterGerrard committed:
Merge pull request #20 from red-gate/fix-findbugs-issues

Fix findbugs issues

PeterLGerrard-Redgate committed:
[maven-release-plugin] prepare release redgate-sql-ci-2.0.0

PeterLGerrard-Redgate committed:
[maven-release-plugin] prepare for next development iteration

PeterLGerrard-Redgate committed:
Ensure we give a non zero exit code when running in PowerShell 3

Commit:
Merge pull request #22 from red-gate/bugfix/SCA-64

Ensure we give a non zero exit code when running in PowerShell 3

Commit:
[maven-release-plugin] prepare release redgate-sql-ci-2.0.1

Commit:
[maven-release-plugin] prepare for next development iteration

PeterLGerrard-Redgate committed:
Add TIL enum

PeterLGerrard-Redgate committed:
Add TIL to build parameters

PeterLGerrard-Redgate committed:
Add Transaction Isolation Level to UI

PeterLGerrard-Redgate committed:
Pretty print in the selection box

PeterLGerrard-Redgate committed:
Address PR comments

Commit:
Merge pull request #23 from red-gate/transaction-isolation-level

Add Transaction isolation level to build step

IvoMiller committed:
Add support for data compare options

IvoMiller committed:
Update SqlCi.ps1

daniel-pottenger committed:
Merge pull request #2 from red-gate/add-data-compare-options

Add data compare options

IvoMiller committed:
Merge remote-tracking branch 'upstream/master'

Commit:
Merge pull request #25 from red-gate/master

Add data compare options

daniel-pottenger committed:
[maven-release-plugin] prepare release redgate-sql-ci-2.0.2

daniel-pottenger committed:
[maven-release-plugin] prepare for next development iteration

daniel-beck-bot committed:
Use HTTPS URLs in pom.xml

diogo-duarte-rg committed:
Use ArgumentBuilder to launch process

Replacing the long string we previously used. This way we can mask the temporaryDatabasePassword parameter

diogo-duarte-rg committed:
Merge pull request #26 from daniel-beck-bot/https-urls-pom

Use HTTPS URLs in pom.xml

diogo-duarte-rg committed:
Merge pull request #3 from red-gate/mask-temporary-db-password

Do not log temporaryDatabasePassword

Commit:
Merge pull request #27 from red-gate/master

Do not log temporaryDatabasePassword

diogo-duarte-rg committed:
[maven-release-plugin] prepare release redgate-sql-ci-2.0.3

diogo-duarte-rg committed:
[maven-release-plugin] prepare for next development iteration

diogo-duarte-rg committed:
[SECURITY-1598]

diogo-duarte-rg committed:
[maven-release-plugin] prepare release redgate-sql-ci-2.0.4

diogo-duarte-rg committed:
[maven-release-plugin] prepare for next development iteration

Commit:
[SECURITY-1696] Encrypt NuGetAPI key

Commit:
[maven-release-plugin] prepare release redgate-sql-ci-2.0.5

Commit:
[maven-release-plugin] prepare for next development iteration

IvoMiller committed:
Update SQL Change Automation templates to enforce TLS 1.2 for PS Gallery

Enforced TLS 1.2 protocol for communication with PowerShell gallery following recent restrictions imposed in PowerShell Gallery.

Commit:
Merge pull request #28 from red-gate/enforce-ps-gallery-protocol

Update SQL Change Automation templates to enforce TLS 1.2 for PS Gallery

Commit:
[maven-release-plugin] prepare release redgate-sql-ci-2.0.6

Commit:
[maven-release-plugin] prepare for next development iteration

BLWood committed:
Configure proxy for certain environment variable

IvoMiller committed:
Update SqlCi.ps1 from SCA main repo

IvoMiller committed:
Add support for connection options

IvoMiller committed:
Improve null safety

IvoMiller committed:
Fix bug

IvoMiller committed:
Merge pull request #29 from red-gate/add-support-for-connection-options

Add support for connection options

IvoMiller committed:
Merge pull request #30 from red-gate/add-proxy-support-for-powershell-gallery

Configure proxy for certain environment variable

IvoMiller committed:
[maven-release-plugin] prepare release redgate-sql-ci-2.0.7

IvoMiller committed:
[maven-release-plugin] prepare for next development iteration

marcin-kaniasty committed:
Added tooltip when creating database package.

marcin-kaniasty committed:
Display tooltips in a style compatible with Jenkins plugins, replacing indent tabs to spaces.

marcin-kaniasty committed:
Merge pull request #31 from red-gate/add_help_tooltip

Added tooltip on Creating database package step.

marcin-kaniasty committed:
[maven-release-plugin] prepare release redgate-sql-ci-2.0.8

marcin-kaniasty committed:
[maven-release-plugin] prepare for next development iteration
